internal/provider/gql.go

```diff
@@ -357,3 +357,86 @@ func (c *ExtendedGqlClient) SetScimGroupMapping(ctx context.Context, orgMrn stri
 
 	return c.Mutate(ctx, &setScimGroupMappingMutation, setScimGroupMappingInput, nil)
 }
+
+func (c *ExtendedGqlClient) UploadComplianceFramework(ctx context.Context, spaceMrn string, content []byte) error {
```

Comment:
Let use just `Framework` instead of `ComplianceFramework`. There are cases where those control frameworks are not a compliance framework but a security framework.